cypress/tsconfig.json

```diff
@@ -1,9 +1,18 @@
 {
     "compilerOptions": {
         "target": "es2016",
-        "lib": ["es2020", "dom"],
+        "jsx": "react",
+        "lib": [
+            "es2020",
+            "dom",
+            "dom.iterable"
+        ],
         "types": ["cypress", "@percy/cypress"],
+        "resolveJsonModule": true,
+        "esModuleInterop": true,
```

Comment:
With all of these changes so far, it's down from 9.5k errors to 175 errors. Now it seems to be down to defining a bunch of globals. What's the smart way to tackle this? Can we inherit from [`src/@types/global.d.ts`](https://github.com/matrix-org/matrix-react-sdk/blob/6574c5c3e2835c6cf25bed58e9e0b62a188dcf72/src/%40types/global.d.ts) which has everything it's complaining about?

Reply:
> Can we inherit from [src/@types/global.d.ts](https://github.com/matrix-org/matrix-react-sdk/blob/6574c5c3e2835c6cf25bed58e9e0b62a188dcf72/src/%40types/global.d.ts) which has everything it's complaining about?

Yes, this is what EW does. https://github.com/vector-im/element-web/blob/develop/src/%40types/global.d.ts#L17